paddlecloud/paddlejob/views.py

```diff
@@ -140,7 +159,7 @@ def post(self, request, format=None):
         except ApiException, e:
             logging.error("error submitting trainer job: %s" % e)
             return utils.simple_response(500, str(e))
-        return utils.simple_response(200, "OK")
+        return utils.simple_response(200, "")
```

Comment:
I've changed paddlecloud client to check if msg is not empty, means there is some error.